From 1d1b233ce7b3275d60b516cb365ae2f34dda57a3 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Thu, 22 Aug 2013 16:14:34 +0800
Subject: [PATCH 240/602] ENGR00276287 Adjust key mapping for 6sl evk_6sl

Add VOLUME_DOWN/VOLUME_UP key mapping for imx-keypad

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 common/input/imx-keypad.kl | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/common/input/imx-keypad.kl b/common/input/imx-keypad.kl
index b2e3ec9..a1600bd 100644
--- a/common/input/imx-keypad.kl
+++ b/common/input/imx-keypad.kl
@@ -8,4 +8,6 @@ key 103    DPAD_UP
 key 105    DPAD_LEFT
 key 106    DPAD_RIGHT
 key 108    DPAD_DOWN
+key 114    VOLUME_DOWN
+key 115    VOLUME_UP
 
-- 
1.8.0

